### ISSUE TITLE
Support the `--directory` option

### ISSUE THREAD

**Issue by deemp** [objectionary/eoc#293]
@maxonfjvipon 

## Problem

Currently, it's unclear where `eoc` looks for files when executing the commands `phi`, `test`, `unphi`, `print`.

## Solution

1. For these commands, add the `--directory` option that accepts a path to a directory.
1. Make `eoc` look for files only in the specified directory.
1. By default (when the `--directory` option is not provided), `eoc` must look for files in the current directory.
1. The default `--directory` option value must be documented in the CLI.
1. `eoc` must report a human-readable error when it found no matching files.
1. `eoc` should never recurse into a `.eoc` directory unless this directory is explicitly specified via `--directory .eoc`.

**Comment by deemp** [comment]
@maxonfjvipon, please document somewhere how to convert PHI to EO and EO to PHI with this option.

**Comment by maxonfjvipon** [comment]
@deemp you can use `eoc phi --help` for getting info about flags. The same story with `unphi` and `print`

**Comment by deemp** [comment]
@maxonfjvipon, there's no info on where it prints the files

```
Usage: eoc print [options]

Generate EO files from XMIR files

Options:
  -?, --help  Print this help information
```

**Comment by maxonfjvipon** [comment]
@deemp it'll be added [here](https://github.com/objectionary/eoc/pull/294)

**Comment by maxonfjvipon** [comment]
@yegor256 let's make a new release with tag `0.21.0`

**Comment by yegor256** [comment]
@rultor release, tag is `0.21.0`

**Comment by yegor256** [comment]
@rultor status

**Comment by yegor256** [comment]
@rultor release, tag is `0.21.0`

**Comment by yegor256** [comment]
@rultor status

**Comment by maxonfjvipon** [comment]
@deemp we released version `0.21.0`. There are new options to specify directories for `phi`, `unphi` and `print` commands. Use `eoc phi --help` to see them
